pkg/services/sqlstore/user.go

```diff
@@ -447,6 +447,17 @@ func SearchUsers(query *m.SearchUsersQuery) error {
 		whereParams = append(whereParams, queryWithWildcards, queryWithWildcards, queryWithWildcards)
 	}
 
+	if query.AuthModule != "" {
+		whereConditions = append(
+			whereConditions, fmt.Sprintf(
+				`id IN (SELECT "user_id"
+				FROM "user_auth"
+				WHERE auth_module='%s')`,
```

Comment:
I think whereParams should be used, using string interpolation for a filter param opens up SQL injection possibilities, maybe not in this case as auth module is a known quantity but still think there is no reason to not add AuthModule to whereParams

Reply:
Good point!